### PR TITLE
docs: Privategpt integration docs.

### PR DESCRIPTION
**Pull request by Anush008**
Section preivew: https://deploy-preview-412--condescending-goldwasser-91acf0.netlify.app/documentation/integrations/privategpt/

### CODE REVIEW

**Comment by Anush008** [comment]
@kacperlukawski, review 🙏.

File: qdrant-landing/content/documentation/integrations/privategpt.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+# PrivateGPT
+
+[PrivateGPT](https://docs.privategpt.dev/) is a production-ready AI project that allows you to ask questions about your documents using the power of Large Language Models (LLMs), even in scenarios without an Internet connection. 100% private, no data leaves your execution environment at any point.
+
+PrivateGPT supports Qdrant as a vectorstore for ingesting and retrieving documents.
+
+## Usage
+
+To enable Qdrant, set the vectorstore.database property in the settings.yaml file to qdrant and install the qdrant extra within your PrivateGPT project.
```
Comment (kacperlukawski) [inline qdrant-landing/content/documentation/integrations/privategpt.md:14]:
```suggestion
To enable Qdrant, install the qdrant extra within your PrivateGPT project first.
```

File: qdrant-landing/content/documentation/integrations/privategpt.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+title: PrivateGPT
+weight: 1600
+---
+
+# PrivateGPT
+
+[PrivateGPT](https://docs.privategpt.dev/) is a production-ready AI project that allows you to ask questions about your documents using the power of Large Language Models (LLMs), even in scenarios without an Internet connection. 100% private, no data leaves your execution environment at any point.
```
[the remark this answers is not in the document]
Comment (Anush008) [reply qdrant-landing/content/documentation/integrations/privategpt.md]: Should be resolved now.
https://github.com/qdrant/landing_page/pull/412/commits/eaac4143bec6f9d4c72dd7178491690d44bddd36.

**Review by kacperlukawski — APPROVED** [review]
LGTM, merging

**Pull request merged**